geode-core/src/test/java/org/apache/geode/internal/cache/tier/sockets/ClientServerMiscDUnitTest.java

```diff
@@ -773,6 +755,73 @@ public void clientIsPreventedFromConnectingToLocatorAsServer() throws Exception
   }
 
 
+  private void proxyRegionClientServerOp(RegionShortcut shortcut) throws Exception {
+    // start server first
+    final String REGION_NAME = "proxyRegionClientServerOp";
+    PORT1 = initServerCache(false);
+    // Create regions on servers.
+    server1.invoke(new SerializableCallable() {
```

Comment:
SerializableRunnable supports "public void run() throws Exception" so there's no need to use SerializableCallable unless you need to return a type. You also don't really need to 1st assertNotNull since c.createRegionFactory will throw NPE:
```Java
     server1.invoke(() -> {
          Cache c = CacheFactory.getAnyInstance();
          Region<Object, Object> r = c.createRegionFactory(shortcut).create(REGION_NAME);
          assertNotNull(r);
      });
```